src/azure-cli/azure/cli/command_modules/storage/tests/latest/test_storage_account_scenarios.py

```diff
@@ -1784,6 +1784,17 @@ def test_storage_account_extended_location(self, resource_group):
         self.cmd('storage container create -n {container} --account-name {sa1} --blob-endpoint {endpoint} --account-key {storage_key}') \
             .assert_with_checks(self.check('created', True))
 
+    @ResourceGroupPreparer(location='eastus2euap')
+    def test_storage_account_migration(self, resource_group):
+        self.kwargs.update({
+            'sa': self.create_random_name('samigration', 24)
+        })
+        self.cmd('az storage account create -n {sa} -g {rg} -l eastus2euap --sku Standard_LRS')
```

Comment:
Why not use `StorageAccountPreparer`? How can we auto-clean the test resource?

Reply:
Changed to use `StorageAccountPreparer`